subsys/metrics/src/main/java/org/commonjava/indy/metrics/reporter/ReporterIntializer.java

```diff
@@ -110,4 +130,82 @@ private void initGraphiteReporterForHealthCheckMetric( MetricRegistry metrics, I
                                                           .build( graphite );
         reporter.start( config.getGrphiterHealthcheckPeriod(), TimeUnit.SECONDS );
     }
+
+    private boolean isExistReporter( String reporter )
+    {
+        return Arrays.asList( config.getReporter().split( ";" ) ).contains( reporter );
```

Comment:
Let's implement this in IndyMetricsConfig, so any component can call `IndyMetricsConfig.isReporterEnabled(<reporter-name>)`. And I think you could just do a `String.contains()` here, right?